nixos/modules/services/hardware/intel-aesmd.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.intel-aesmd;
+
+in {
+
+  options.services.intel-aesmd = {
+    enable = mkOption {
```

Comment:
On this option I'm conflicted, because it really is intel specific, so it would fit into `hardware.cpu.intel`, but it's also a service, and we put all services into `services.*`. I'm leaning more towards the former though, because `services.*` is already big enough and I'd feel like people would rather search for `intel` to find this service than looking through `services.*`

Reply:
I moved all the options to hardware.cpu.intel and renamed `intel-microcode.nix` to `intel.nix`